Sources/Tracking/CustomerIOImplementation.swift

```diff
@@ -72,93 +65,19 @@ class CustomerIOImplementation: CustomerIOInstance {
+        DataPipeline.shared.identify(body: body)
     }
 
-    func handleCommonIdentificationTasks(identifier: String, dictionaryBody: [String: Any]? = nil, codableBody: Codable? = nil) {
-        if identifier.isBlankOrEmpty() {
-            logger.error("profile cannot be identified: Identifier is empty. Please retry with a valid, non-empty identifier.")
-            return
-        }
-
-        // Check which body is non-nil and proceed accordingly
-        if let body = dictionaryBody {
-            DataPipeline.shared.identify(identifier: identifier, body: body)
-        } else if let body = codableBody {
-            DataPipeline.shared.identify(identifier: identifier, body: body)
-        } else {
-            DataPipeline.shared.identify(identifier: identifier)
-        }
-
-        let currentlyIdentifiedProfileIdentifier = profileStore.identifier
-        let isChangingIdentifiedProfile = currentlyIdentifiedProfileIdentifier != nil &&
-            currentlyIdentifiedProfileIdentifier != identifier
-        let isFirstTimeIdentifying = currentlyIdentifiedProfileIdentifier == nil
-
-        if let currentlyIdentifiedProfileIdentifier = currentlyIdentifiedProfileIdentifier,
-           isChangingIdentifiedProfile {
-            logger.info("changing profile from id \(currentlyIdentifiedProfileIdentifier) to \(identifier)")
-
-            logger
-                .debug(
-                    "deleting token from previously identified profile to prevent sending messages to it. It's assumed that for privacy and messaging relevance, you only want to send messages to devices that a profile is currently identifed with."
-                )
-            deleteDeviceToken()
-
-            logger.debug("running hooks changing profile from \(currentlyIdentifiedProfileIdentifier) to \(identifier)")
-            hooks.profileIdentifyHooks.forEach { hook in
-                hook.beforeIdentifiedProfileChange(
-                    oldIdentifier: currentlyIdentifiedProfileIdentifier,
-                    newIdentifier: identifier
-                )
-            }
-        }
-
-        logger.debug("storing identifier on device storage \(identifier)")
-        profileStore.identifier = identifier
-
-        if isFirstTimeIdentifying || isChangingIdentifiedProfile {
-            if let existingDeviceToken = globalDataStore.pushDeviceToken {
-                logger.debug("registering existing device token to newly identified profile: \(identifier)")
-                // this code assumes that the newly identified profile has been saved to device storage. only call this
-                // function until after the SDK stores the new profile identifier
-                registerDeviceToken(existingDeviceToken)
-            }
-
-            logger.debug("running hooks profile identified \(identifier)")
-            hooks.profileIdentifyHooks.forEach { hook in
-                hook.profileIdentified(identifier: identifier)
-            }
-        }
```

Comment:
I have removed all these checks because it looks like they aren't needed for `analytics`